Modern UI

Bashamega opened the pull request.

pvorb commented:
Could you please explain what you did and why?

pvorb commented:
It's interesting how you invested time into the redesign, but didn't bother to explain what you did.

Bashamega commented:
> It's interesting how you invested time into the redesign, but didn't bother to explain what you did.

Sorry I forgot, and didn't see your first reply.
I have deleted the repo so explaining won't do any good. 
But the conclusion is I have updated the packages because they were outdated.
Fix webpack configuration to work with the latest version.
Updated the design because the current one looks bad
